src/SingleBookingPage/bookingItem/AccordionBodyLeg.js

```diff
@@ -71,20 +71,20 @@ const AccordionBodyLeg = (props: LegProps) => {
     <React.Fragment>
       <div className="leg">
         <div className="calendarIcon">
-          <Calendar customColor="#2f363c" height="20" />
+          <Calendar customColor="#2f363c" size="medium" />
         </div>
         <div className="date">
           {FormatDate({ dateString: initialDeparture })}
         </div>
         <AccordionLegCities leg={props.leg} />
         <div className="layover">
-          <Clock customColor="#7c8b99" height="16" />
+          <Clock customColor="#7c8b99" size="small" />
```

Comment:
this size prop should be also medium, now it's breaking the alignment of the text

![screen shot 2018-08-17 at 18 16 58](https://user-images.githubusercontent.com/18072606/44277069-c8348e80-a249-11e8-95a2-2c7a7bee53bd.png)